src/main/java/frc/team2412/robot/commands/climb/ClimbStageChooserCommand.java

```diff
@@ -0,0 +1,34 @@
+package frc.team2412.robot.commands.climb;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.ClimbSubsystem;
+
+public class ClimbStageChooserCommand extends CommandBase {
+
+    private ClimbSubsystem subsystem;
+
+    public ClimbStageChooserCommand(ClimbSubsystem subsystem) {
+        this.subsystem = subsystem;
+    }
+
+    @Override
+    public void initialize() {
+        switch (subsystem.getAutoClimbState()) {
```

Comment:
For the first competition, should there be a low rung climb instead of the traverse climb?  While mid will work for low, the use case for a low rung bar is that we continue shooting to the very end and then climb the low bar at the very last second so wasting time extending way past the bar height may not be advantageous.